test/testsuites/operatorhubtests.go

```diff
@@ -213,6 +214,62 @@ func testDisableNonDefault(t *testing.T) {
 	resetClusterOperatorHub(t, namespace)
 }
 
+// testClusterStatusDefaultsDisabled tests that, when all default operator sources are disabled,
+// the clusterstatus sets Available=True
+func testClusterStatusDefaultsDisabled(t *testing.T) {
```

Comment:
Can you make sure there's a test that covers the case where they're not all disabled? I would've assumed there was one already, but if so it should've caught the issue with `func (o *operatorhub) Disabled()`

Reply:
The test where they are not all disabled should be the same as the test where the operatorhub isn't set. We can definitely add a test explicitly checking for that though.